tests/e2e/actions.go

```diff
@@ -1795,15 +1800,28 @@ func (tr TestRun) invokeDoublesignSlash(
 	action doublesignSlashAction,
 	verbose bool,
 ) {
-	chainConfig := tr.chainConfigs[action.chain]
-	//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
-	bz, err := exec.Command("docker", "exec", tr.containerConfig.instanceName, "/bin/bash",
-		"/testnet-scripts/cause-doublesign.sh", chainConfig.binaryName, string(action.validator),
-		string(chainConfig.chainId), chainConfig.ipPrefix).CombinedOutput()
-	if err != nil {
-		log.Fatal(err, "\n", string(bz))
+	if !tr.useCometmock {
+		chainConfig := tr.chainConfigs[action.chain]
+		//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
+		bz, err := exec.Command("docker", "exec", tr.containerConfig.instanceName, "/bin/bash",
+			"/testnet-scripts/cause-doublesign.sh", chainConfig.binaryName, string(action.validator),
+			string(chainConfig.chainId), chainConfig.ipPrefix).CombinedOutput()
+		if err != nil {
+			log.Fatal(err, "\n", string(bz))
+		}
+		tr.waitBlocks("provi", 10, 2*time.Minute)
+	} else { // tr.useCometMock
+		validatorAddress := tr.GetValidatorAddress(action.chain, action.validator)
+
+		method := "cause_double_sign"
```

Comment:
Curious where the script lives that actually handles `"cause_double_sign"`?

Reply:
it's a cometmock RPC method, it does not live here. It's a feature of cometmock